src/main/java/com/stripe/model/issuing/Card.java

```diff
@@ -122,6 +128,9 @@
   @SerializedName("shipping")
   Shipping shipping;
 
+  @SerializedName("spending_controls")
+  AuthorizationControls spendingControls;
```

Comment:
The class name **should** be `SpendingControls` but internally it's shared as one resource with `AuthorizationControls`.

While it's not great, we're doing a fast follow with a major version that removes all deprecated features and that major version will rename that class too.